AndroidSDK/src/com/leanplum/LeanplumNotificationChannel.java

```diff
@@ -602,10 +604,14 @@ private static int getTargetSdkVersion(Context context) {
       showBadge = (boolean) CollectionUtil.getOrDefault(channel, "show_badge", showBadge);
 
       try {
-        List<Long> pattern = CollectionUtil.uncheckedCast(
+        List<Number> pattern = CollectionUtil.uncheckedCast(
             CollectionUtil.getOrDefault(channel, "vibration_pattern", null));
```

Comment:
Are you sure this will work?
in case of default value wouldn't it be:
`List<Number> pattern = (List<Number>)"vibrationPattern"` which is trying to cast a string to a list?